docs/getting-started.md

```diff
@@ -28,3 +32,15 @@ curl -fsSL https://raw.githubusercontent.com/spicetify/spicetify-marketplace/mai
 - ⚠️ Sometimes Spotify upgrades things that break Spicetify.<br/>
 - ⚠️ When this happens we need to update Spicetify, and you may need to run `spicetify upgrade`.<br/>
 - ⚠️ After this command, run `spicetify restore backup apply` to have everything back where it was.
+
+## Basic Usage
+
+After installing Spicetify, and the Spicetify's Marketplace, you can now use it to customize your Spotify client using all the available **extensions** and **themes** found in the Marketplace.
+
+### Updating
```

Comment:
This section and the warnings above basically repeat each other.
In my opinion, this section should replace the warnings completely as it is written as a paragraph instead of a list and explains things more in-depth.